sklearn/utils/estimator_checks.py

```diff
@@ -2591,14 +2555,10 @@ def check_estimators_data_not_an_array(name, estimator_orig, X, y, obj_type):
 
 
 def check_parameters_default_constructible(name, Estimator):
```

Comment:
Can we rename it to `estimator` or do you think it would be considered an API breakage?

Reply:
yeah it's not ideal but I'm concerned about backward compat :/